### PR TITLE
in connection with the addition of the profile of the k1 max.

### PR DESCRIPTION
**Pull request by vlotho**

### CODE REVIEW

File: resources/profiles/Creality/filament/Generic PLA @Creality K1 max.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "type": "filament",
+    "filament_id": "GFL99",
+    "setting_id": "GFSL99_CREALITY_00",
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/filament/Generic PLA @Creality K1 max.json:4]: The setting_id is duplicated. Use GFSL99_CREALITY_01 instead

File: resources/profiles/Creality/filament/Hyper PLA @Creality K1 max.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    "bed_temperature_difference": [
+        "10"
+    ],
+    "chamber_temperature": "0",
+    "close_fan_the_first_x_layers": [
+        "1"
+    ],
+    "compatible_printers": [
+        "MyKlipper 0.4 nozzle",
+        "MyMarlin 0.4 nozzle"
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/filament/Hyper PLA @Creality K1 max.json:14]: We do not have these printers in profiles

File: resources/profiles/Creality/filament/Hyper PLA @Creality K1 max.json [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    "temperature_vitrification": [
+        "60"
+    ],
+    "textured_plate_temp": [
+        "60"
+    ],
+    "textured_plate_temp_initial_layer": [
+        "60"
+    ],
+    "version": "1.6.0.0"
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/filament/Hyper PLA @Creality K1 max.json:184]: We do not have key "version"

File: resources/profiles/Creality/machine/Creality K1 Max 0.4 nozzle.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+  "type": "machine",
+  "setting_id": "GM_CREALITY_000",
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/machine/Creality K1 Max 0.4 nozzle.json:3]: The setting_id is duplicated. Use GM_CREALITY_018 instead.

File: resources/profiles/Creality/process/0.20mm Quality @Creality K1 Max 0.4.json [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+    "bridge_angle": "0",
+    "bridge_density": "100%",
+    "bridge_flow": "0.95",
+    "bridge_no_support": "0",
+    "bridge_speed": "50",
+    "brim_object_gap": "0.1",
+    "brim_type": "auto_brim",
+    "brim_width": "5",
+    "compatible_printers": [
+        "MyKlipper 0.4 nozzle"
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/process/0.20mm Quality @Creality K1 Max 0.4.json:18]: The same problem

File: resources/profiles/Creality/process/0.20mm Quality @Creality K1 Max 0.4.json [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+    "gcode_comments": "0",
+    "gcode_label_objects": "1",
+    "independent_support_layer_height": "1",
+    "infill_anchor": "400%",
+    "infill_anchor_max": "20",
+    "infill_combination": "0",
+    "infill_direction": "45",
+    "infill_jerk": "9",
+    "infill_wall_overlap": "25%",
+    "inherits": "0.20mm Standard @MyKlipper",
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/process/0.20mm Quality @Creality K1 Max 0.4.json:55]: Can't find the profile

File: resources/profiles/Creality/process/0.20mm Standard @Creality K1 Max 0.4.json [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+    "bridge_angle": "0",
+    "bridge_density": "100%",
+    "bridge_flow": "0.95",
+    "bridge_no_support": "0",
+    "bridge_speed": "50",
+    "brim_object_gap": "0.1",
+    "brim_type": "auto_brim",
+    "brim_width": "5",
+    "compatible_printers": [
+        "MyKlipper 0.4 nozzle"
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/process/0.20mm Standard @Creality K1 Max 0.4.json:18]: The same problem

File: resources/profiles/Creality/process/0.20mm Standard @Creality K1 Max 0.4.json [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+    "gcode_comments": "0",
+    "gcode_label_objects": "1",
+    "independent_support_layer_height": "1",
+    "infill_anchor": "400%",
+    "infill_anchor_max": "20",
+    "infill_combination": "0",
+    "infill_direction": "45",
+    "infill_jerk": "9",
+    "infill_wall_overlap": "25%",
+    "inherits": "0.20mm Standard @MyKlipper",
```
Comment (XunZhangBambu) [inline resources/profiles/Creality/process/0.20mm Standard @Creality K1 Max 0.4.json:55]: The same

**Comment by iZonex** [comment]
@lanewei120 Please close this PR (https://github.com/bambulab/BambuStudio/pull/2914) Already in master branch.

**Pull request closed without merging**